asp_interactive_configuration/DOC.md

```diff
@@ -130,17 +138,72 @@ The user input corresponds to the current partial configuration `C` that is bein
 
 **Externals**
 
-- **`user(ooasp_isa_leaf(CONFIG,LEAFCLASS,ID))`**
-  The user assigned leaf class `LEAFCLASS` to object `ID`
+- **`user(ooasp_isa(CLASS,ID))`**
+  The user assigned class `CLASS` to object `ID`
 
-- **`user(ooasp_associated(CONFIG,ASSOC,ID1,ID2))`**
-  The user associated `ID1` and `ID2`
+- **`user(ooasp_associated(ASSOC,ID1,ID2))`**
+  The user associated `ID1` and `ID2` via `ASSOC`
 
-- **`user(ooasp_attribute_value(CONFIG,N,ID,VALUE))`**
+- **`user(ooasp_attribute_value(N,ID,VALUE))`**
   The user selected value `VALUE` for attribute `N` of object `ID`
 
 The truth value of these externals is defined by the [Interactive Configurator](#interactive-configurator) based on the current (partial) configuration that is being constructed using an object of class [OOASPConfiguration](#configuration).
 
+The user externals are generated restricted by the possible classes that an object can have. This is defined by the argument `cls` passed in the grounding, which corresponds to the domain defined by `ooasp_domain(_,cls).`. Therefore, if the new object is grounded for a class `cls`, only valid attributes and associations for `cls` and its subclasses will be considered. This is summarized in the auxiliary predicate `counts_as(O,C)`, which states that object `O` can count as class `C`.
+
+
+## Numerical attributes
+
+Numerical attributes are treated using the system `fclingo`. This systems adds a founded condition while using `clorm` in the background. 
+Attributes whose type is `int`, such as `nr_passengers`, `standing_room`, and `nr_seats` in the Metro example, are treated as variables with this system as explained below. Note that numerical values can also be treated as an enumeration using the type `enumint` instead. This option will consider all values in the defined range for predicate `ooasp_attr_value`, such as `frame_position` in the racks example.
+
+### Usage
+
+When an attribute `ATTR` is of type `int`, their value is defined the `fclingo` variable `ooasp_attr_fvalue(ATTR,O)`, which will be associated to the value of `ATTR` for object `O`.
```

Comment:
This sentence is grammatically incorrect. I don't know how to fix it because I'm not entirely sure what it means.